pkg/assets/builder.go

```diff
@@ -24,6 +24,8 @@ import (
 	"strings"
 )
 
+var YAMLSep = []byte("\n\n---\n\n")
```

Comment:
I was thinking that it would be more performant.  But I will move it inside the `func`